composer.json

```diff
@@ -14,13 +14,15 @@
     "minimum-stability": "dev",
     "require": {
         "php": "^7.0.8",
+        "doctrine/annotations": "^1.11@dev",
```

Comment:
I think this may have been added by accident. But I could be mistaken. Either way, requiring `@dev` would be wise here, as app in production would then be forced to use unstable branches.  Same for `symfony/flex`

Reply:
Yes, sorry. i am looking for fixing all theses errors, it is my first contribution to a collectif project i am a bit lost...

Reply:
hey no problem, if you have any questions feel free to DM on slack `jrushlow`